commitizen/cz/base.py

```diff
@@ -95,6 +96,46 @@ def schema_pattern(self) -> str | None:
         """Regex matching the schema used for message validation."""
         raise NotImplementedError("Not Implemented yet")
 
+    def validate_commit_message(
+        self,
```

Comment:
```suggestion
        self,
        *,
```

Let's make it keyword only args to avoid passing args in the wrong order

Reply:
This is fixed now.